guides/manual-installation.md

```diff
@@ -340,364 +304,323 @@ journalctl -fu trento-web
 
-Enable and start Docker:
-
-```bash
-systemctl enable --now docker
-```
+    ```bash
+    systemctl enable --now docker
+    ```
 
 #### Create a dedicated docker network for Trento
 
-```bash
-docker network create trento-net
-```
+1. Create trento docker network:
+    ```bash
+    docker network create trento-net
+    ```
 
-> Note: When creating the trento-net network, Docker typically assigns a default subnet: `172.17.0.0/16`. Ensure that this subnet matches the one specified in your PostgreSQL configuration file, which can be found at `/var/lib/pgsql/data/pg_hba.conf`. If the subnet of `trento-net` differs from `172.17.0.0/16` then adjust `pg_hba.conf` and restart postgresql.
+    > **Note:** When creating the trento-net network, Docker typically assigns a default subnet: `172.17.0.0/16`. Ensure that this subnet matches the one specified in your PostgreSQL configuration file, which can be found at `/var/lib/pgsql/data/pg_hba.conf`. If the subnet of `trento-net` differs from `172.17.0.0/16` then adjust `pg_hba.conf` and restart postgresql.
 
-Verifying the subnet of trento-net:
+1. Verifying the subnet of trento-net:
 
-```bash
-docker network inspect trento-net  --format '{{range .IPAM.Config}}{{.Subnet}}{{end}}'
-```
+    ```bash
+    docker network inspect trento-net  --format '{{range .IPAM.Config}}{{.Subnet}}{{end}}'
+    ```
 
-Expected output:
+    Expected output:
 
-```
-172.17.0.0/16
-```
+    ```bash
+    172.17.0.0/16
+    ```
 
 #### Install Trento on docker
 
-> Note: The environment variables listed here are examples and should be considered as placeholders. Instead of specifying environment variables directly in the docker run command, it is recommended to use an environment variable file. Store your environment variables in a file and use the --env-file option with the docker run command. Find detailed instructions on how to use an environment variable file with Docker on [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env).
-
-##### Create the secret environment variables
-
-```bash
-WANDA_SECRET_KEY_BASE=$(openssl rand -out /dev/stdout 48 | base64)
-TRENTO_SECRET_KEY_BASE=$(openssl rand -out /dev/stdout 48 | base64)
-ACCESS_TOKEN_ENC_SECRET=$(openssl rand -out /dev/stdout 48 | base64)
-REFRESH_TOKEN_ENC_SECRET=$(openssl rand -out /dev/stdout 48 | base64)
-```
-
-##### Install trento-wanda on docker
-
-```bash
-docker run -d --name wanda \
-    -p 4001:4000 \
+The environment variables listed here are examples and should be considered as placeholders. Instead of specifying environment variables directly in the docker run command, it is recommended to use an environment variable file. Store your environment variables in a file and use the --env-file option with the docker run command. Find detailed instructions on how to use an environment variable file with Docker on [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env).
```

Comment:
```suggestion
The environment variables listed here are examples and should be considered as placeholders. Instead of specifying environment variables directly in the `docker run` command, it is recommended to use an environment variable file. Store your environment variables in a file and use the --env-file option with the `docker run` command. Find detailed instructions on how to use an environment variable file with Docker on [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env).
```